release notes/v0.46.0.md

```diff
@@ -0,0 +1,185 @@
+The browser HTTP metrics have also been modified, as these are now split from the HTTP module ones, so there are new `browser_` prefixed HTTP metrics, specifically for request duration and failed requests ([browser#916](https://github.com/grafana/xk6-browser/pull/916)).
+
+Another big change introduced in this version affects the way the **browser lifecycle** is handled. Users no longer have to explicitly initialize/close the browser instance through the JS API. Instead, a browser instance will be automatically initialized/closed at the beginning/end of each iteration if the browser type is set in scenario options (see [#3036](https://github.com/grafana/k6/pull/3036)). This also means that the `chromium` entity from `k6/experimental/browser` import path is no longer valid. Instead, the `browser` entity provides access to the browser methods such as `browser.newPage()` ([browser#910](https://github.com/grafana/xk6-browser/pull/910), [browser#944](https://github.com/grafana/xk6-browser/pull/944)). This change means that the `browser.on()` method is no longer applicable, and therefore it has been removed ([browser#919](https://github.com/grafana/xk6-browser/pull/919)).
+
+Additionally, related to **import changes**, the browser module version is no longer an exported field ([browser#923](https://github.com/grafana/xk6-browser/pull/923)).
+
+Last but not least, this release also includes constraints on **browser contexts** usage as now only one `browserContext` is allowed per iteration, which means that the user can create a new `browserContext`, close it, and then create it again; but can not have more than one "live" `browserContext`. Instead, [scenarios](https://k6.io/docs/using-k6/scenarios/) should be used to separate independent actions in a test ([browser#929](https://github.com/grafana/xk6-browser/pull/929), [browser#945](https://github.com/grafana/xk6-browser/pull/945)).
+
+With all these changes, a simple browser test now looks like:
+```js
+import { browser } from 'k6/experimental/browser';
+
+export const options = {
+  scenarios: {
+    ui: {
+      executor: 'shared-iterations',
+      options: {
+        browser: {
+          type: 'chromium', // chromium is the only supported browser type so as long as
+                            // the option is set, Chromium/Google Chrome will be used
+        },
+      },
+    },
+  },
+};
+
+export default async function () {
+  const page = browser.newPage();
+  try {
+    await page.goto('https://grafana.com')
+  } finally {
+    page.close();
+  }
+}
+```
+
+### Deprecations
+
+- [#3121](https://github.com/grafana/k6/issues/3121) deprecates the `loadimpact/k6` docker image. We intend to keep supporting it for at most a year from now, but we recommend switching to the `grafana/k6` image as soon as possible.
```

Comment:
```suggestion
- [#3121](https://github.com/grafana/k6/issues/3121) deprecates the `loadimpact/k6` docker image. We recommend switching to the `grafana/k6` image as soon as possible.
```

It does not match what we are reporting in the docker hub page:

>THIS IMAGE IS DEPRECATED and its support will be discontinued after Dec 31, 2023. This image has been replaced by [grafana/k6](https://hub.docker.com/r/grafana/k6).

Reply:
Fair enough, I used what was mentioned in the PR initially 👍🏻